lib/commit-lint.ts

```diff
@@ -24,21 +24,19 @@ export class LintCommit {
      * Linter method to run checks on the commit message.
      * @param {IPRCommit} the patch to be checked
      *
-     * The lintings are in methods called from here.  If
+     * The lints are in methods called from here.  If
      * a lint check is too severe to continue, it will throw
      * an error.
      */
-    public async lint(): Promise<ILintError | void> {
+    public lint(): ILintError | void {
```

Comment:
Doesn't CIHelper::checkCommits expect a promise from this?

Reply:
Hmm. In `checkCommits()`, we pass this through `Promise.all()`, right? According to [the documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all) it is totally legitimate to pass non-`Promise`s to `Promise.all()`:

```js
[...]
const promise2 = 42;
[...]
Promise.all([promise1, promise2, promise3]).then([...])
```